gtsam/gtsam.i

```diff
@@ -977,6 +977,52 @@ class Cal3Bundler {
   void pickle() const;
 };
 
+#include <gtsam/geometry/Cal3Fisheye.h>
+class Cal3Fisheye {
```

Comment:
should this be a virtual class inheriting from a base class?

Reply:
Yeah this should technically inherit from `Cal3` (similar case for the other camera classes). This is my bad when I refactored the camera classes. :slightly_frowning_face: